src/test/java/edu/oakland/test/display02/DisplayCommManagerImplementationTests.java

```diff
@@ -0,0 +1,40 @@
+package edu.oakland.test.display02;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import edu.oakland.helper.admin.LocationDataPoint;
+import edu.oakland.helper.admin.TrackData;
+import edu.oakland.production.display02.DisplayCommManager;
+import edu.oakland.production.display02.DisplayCommManagerImplementation;
+import edu.oakland.production.middleware02.MiddlewareCommInterface;
+import edu.oakland.test.display02.MiddlewareCommInterfaceStub;
+import java.lang.IllegalArgumentException;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+@DisplayName("Display Comm Manager Implementation Tests")
+public class DisplayCommManagerImplementationTests {
+
+  @Test
+  @DisplayName("RFID In Is the same as Rfid Out")
+  void rfidInIsRfidOut() {
+    int rfid = generateRandomRfid();
+    MiddlewareCommInterfaceStub testStub = new MiddlewareCommInterfaceStub();
+    DisplayCommManagerImplementation displayComm = new DisplayCommManagerImplementation(testStub);
+    displayComm.receiveRfid(rfid);
+    assertEquals(rfid, displayComm.rfid);

```

Comment:
The DisplayCommManager does not store the RFID, so what you want to do is verify that the returning `TrackData` has the right RFID number in the message.
```suggestion
    assertEquals(rfid, Integer.parseInt(displayComm.receiveRfid(rfid).getStatusMessage()));

```